src/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.0.4 on 2024-05-03 07:27
```

Comment:
These migrations are auto generated by django as a record for when the schema changes, I was told to commit them so 🤷, I assume no one will need another migration either way